bioblend/galaxy/folders/__init__.py

```diff
@@ -56,11 +64,61 @@ def show_folder(self, folder_id: str, contents: bool = False) -> Dict[str, Any]:
         :param contents: True to get the contents of the folder, rather
           than just the folder details.
 
+        :type limit: int
+        :param limit: Maximum number of contents to return (default: 10).
+          Only considered for contents=True.
+
+        :type offset: int
+        :param contents: Return contents from this specified position (default: 0).
+          Only considered for contents=True.
+
+        :type include_deleted: bool
+        :param include_deleted: Returns also deleted contents.
+          Only considered for contents=True.
+
         :rtype: dict
         :return: dictionary including details of the folder
         """
+        params = {
+            "limit": limit,
+            "offset": offset,
+            "include_deleted": include_deleted,
+        }
+        return self._get(id=folder_id, contents=contents, params=params)
+
+    def contents(
+        self,
+        folder_id: str,
+        limit: int = 10,
```

Comment:
I'd call it `chunk_size` in this context, as used in the rest of the library.

Reply:
I can rename this, but actually it seems that we already have a mix of limit and chunk_size (slightly more limit .. it seems).